packages/amplify_authenticator/lib/src/widgets/default_forms.dart

```diff
@@ -5,56 +5,80 @@ import 'package:amplify_authenticator/src/widgets/form_fields.dart';
 
 class DefaultForms {
   static SignInForm signInForm() {
-    return SignInForm(
+
+    return const SignInForm(
+
       formFields: FormFields(
         children: [
           SignInFormField(
               title: 'Username*',
-              hintText: "Enter you username",
+              hintText: "Enter your username",
               type: 'username'),
           SignInFormField(
               title: 'Password*',
-              hintText: "Enter you password",
+              hintText: "Enter your password",
+
               type: 'password')
         ],
       ),
     );
   }
 
   static SignUpForm signUpForm() {
-    return SignUpForm(
+
+    return const SignUpForm(
+
       formFields: FormFields(
         children: [
           SignUpFormField(
               title: 'Username*',
-              hintText: "Enter you username",
+
+              hintText: "Enter your username",
               type: 'username'),
           SignUpFormField(
               title: 'Password*',
-              hintText: "Enter you password",
+              hintText: "Enter your password",
               type: 'password'),
           SignUpFormField(
-              title: 'Email*', hintText: "Enter you email", type: 'email'),
+              title: 'Email*', hintText: "Enter your email", type: 'email'),
           SignUpFormField(
               title: 'Phone Number*',
-              hintText: "Enter you phone number",
+              hintText: "Enter your phone number",
+
               type: 'phone_number')
         ],
       ),
     );
   }
 
   static ConfirmSignUpForm confirmSignUpForm() {
-    return ConfirmSignUpForm(
+
+    return const ConfirmSignUpForm(
+
       formFields: FormFields(
         children: [
           ConfirmSignUpFormField(
               title: 'Username*',
-              hintText: "Enter you username",
+
+              hintText: "Enter your username",
               type: 'username'),
           ConfirmSignUpFormField(
               title: 'Verification Code*',
-              hintText: "Enter you verification code",
+              hintText: "Enter your verification code",
+              type: 'code'),
+        ],
+      ),
+    );
+  }
+
+  static ConfirmSignInForm confirmSignInForm() {
+    return const ConfirmSignInForm(
+      formFields: FormFields(
+        children: [
+          ConfirmSignInFormField(
+              title: 'Code*',
+              hintText: "Enter your confirmation code",
```

Comment:
Interesting edge case I bumped into today on discord: so, if a user is created by an admin in Cognito, they issue a temporary password. When the user logs in for the first time with the temp password - the signInStep will be `CONFIRM_SIGN_IN_WITH_NEW_PASSWORD` and users should be prompted for a new password, after which calling  `Amplify.Auth.confirmSignIn(confirmationValue: newPassword);` saves the new password and logs them in.

Now, let's say that MFA is also enabled. The signInStep from *that* call will be `CONFIRM_SIGN_IN_WITH_SMS_MFA_CODE`. And finally after calling `confirmSignIn` again with the MFA code, the signInStep will be `DONE`. But they will need to be presented the confirm sign in screen twice. And on the first time, we probably want to show "Enter a new password" or something to that effect instead of "Enter your confirmation code"

Reply:
@israx -  @dnys1 's comment above is what I mentioned on Tuesday. You can handle it in a separate PR if you want.

Reply:
Cool cool. Approving it now so @israx you can handle how you think best.